sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -355,6 +359,7 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
 
   @UiThread
   fun onSurfaceDestroyed() {
+    logW(TAG, "RenderThread : surface destroyed")
```

Comment:
nit: We are adding too many logs imo, and these shouldn't be warning messages.

Reply:
personally I agree on leaving logs related to surface / activity lifecycle here as those events are pretty rare but at the same time will give us pretty much info in case of some rendering issues.

Reply:
@pengdev I've switched all new logs to INFO, and @kiryldz is right about the intentions - debug will be much easier while logs only happen on map creation/destruction or pauses, which is pretty occasional